lisa/features/resize.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from enum import Enum
+from typing import TYPE_CHECKING
+
+from lisa.feature import Feature
+
+FEATURE_NAME_RESIZE = "Resize"
+
+
+class ResizeAction(str, Enum):
+    IncreaseCpuCount = "IncreaseCpuCount"
```

Comment:
@squirrelsc can we change the names of these to IncreaseCoreCount and DecreaseCoreCount since we're using `candidate_size.capability.core_count` in features.py? I think it would be better for consistency.

Reply:
Sure, please change it to keep consistent.

Reply:
done

Reply:
Please let me resolve comments, so I can track their progress.